src/sass/layout/_pokemonField.scss

```diff
@@ -0,0 +1,7 @@
+.pokemonField{
```

Comment:
in sass or css isn't neccesary to use camelCase I personally do, but here if you will always use BEM convention or any convention be consistent not use it in some places and in other doesn't.